### ISSUE TITLE
Does mc mirror have a Python sdk？

### ISSUE THREAD

**Issue by xujuchao** [minio/minio-py#1359]
I want to backup data to aws through Python sdk. Is there an sdk similar to mc mirror?

**Comment by balamurugana** [comment]
No. You would need to write the code with  `list_object()` and `fget_object()` APIs.
